zach.js

```diff
@@ -1,113 +1,55 @@
-
- //Airbnb API
- const optionsA = {
-
+//Airbnb API
 
```

Comment:
This is what I changed  there were two const optionsA which was  throwing an error.